### DISCUSSION TITLE
Possible bug : color brush uses light theme only.

### DISCUSSION THREAD

**Discussion by Al-Dyachkov** [irihitech/Semi.Avalonia#363] [Q&A] [answered]
I have a path that is filled using Fill="{DynamicResource SemiGrey7}"
When using light theme color is 41464c, as expected, it matches light theme brush.
When using dark theme color is still 41464c, and not C6CACD that should be used according to demo.

Brushes like SemiColorWarning work as expected.
Am I missing something ?

**Comment by rabbitism** [comment]
hmm, I cannot reproduce

**Reply by Al-Dyachkov** [reply]
Can you give me any tip on troubleshooting this ? Property value is local, it is not overriden by any style etc.

**Reply by Al-Dyachkov** [reply, marked as answer]
Never mind.
Looks like my local repo of semi got corrupted somehow (I'm building from source to have latest). Clean download showed no signs of this issue in demo and rebuild with clean sources fixed the issue in my app.
